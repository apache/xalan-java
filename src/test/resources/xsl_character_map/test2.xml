<root imp_attr="1, 2 and 3">
  <a id="1">hello 1</a>
  <b id="2">hello 2</b>
  <c id="3">
    <d>hello 3</d>
  </c>
</root>
