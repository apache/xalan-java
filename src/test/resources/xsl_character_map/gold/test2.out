<?xml version="1.0" encoding="UTF-8"?><result>
  <one>thanks hello1 for reading hello2</one>
</result>
